changelogs/fragments/62790-openssl_certificate_fix_assert.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- "openssl_certificate - fix assertonly provider certificate verification, causing 'private key mismatch' and 'subject mismatch' errors."
```

Comment:
```suggestion
- "openssl_certificate - fix ``assertonly`` provider certificate verification, causing 'private key mismatch' and 'subject mismatch' errors."
```

Reply:
done, sorry to not have pushed it earlier

Reply:
No problem, thanks for pushing it! :)